### DISCUSSION TITLE
use within a terraform step

### DISCUSSION THREAD

**Discussion by mattshort77** [Azure/secrets-store-csi-driver-provider-azure#485] [Q&A]
hi

I've tested this out by just following the instructions and managed to prove that it does what i want. 

but i'd prefer to deploy by terraform. 
I can't quite get the chart and repository combo right, has anyone already done this and worked that out?
```
  repository = "https://github.com/Azure/secrets-store-csi-driver-provider-azure/master/charts"
  chart      = "csi-secrets-store-provider-azure/csi-secrets-store-provider-azure"
```
from the examples on this page https://registry.terraform.io/providers/hashicorp/helm/latest/docs

thanks,

**Comment by mattduguid** [comment]
@mattshort77 this works for us,
  
```
  chart = "csi-secrets-store-provider-azure"
  repository = "https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/charts/"
```
